tests/bootstrap.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Tests;
+
+use Dotenv\Dotenv;
+use LiveIntent\Services\Concerns\MocksRequests;
+
+require_once(dirname(__FILE__) . '/../vendor/autoload.php');
+
+class TestBootStrap
+{
+    use MocksRequests;
```

Comment:
Similar to the above comment, the MocksRequests trait should be used in the test class (can be parent) but not the bootstrap